### DISCUSSION TITLE
Download collected data

### DISCUSSION THREAD

**Discussion by filiphanes** [plausible/analytics#648] [Anything Else]
Hi,
I would like to download collected data: pageviews, ... in csv for migration or just for spreadsheet analytics.
Is it possible? Thanks

**Comment by metmarkosaric** [comment]
hi @filiphanes! you can download the number of daily visitors by following these instructions: https://plausible.io/docs/export-stats

full data import / export is not possible at this time unfortunately but we'd like to add it. at the moment our development focus is on the API as that's the most requested feature but we will probably work on import / export as the next big feature after the API. you can read more here and follow our progress #211

**Comment by metmarkosaric** [comment]
You can now export all of your stats in a CSV format. Click on the download icon on the top right of your Plausible dashboard. It will export a ZIP file which includes CSV files of the individual reports. You can then import your stats into Excel or other data analysis tools. Learn more here: https://plausible.io/docs/export-stats

**Reply by webtail** [reply]
@metmarkosaric Any workaround for the pages export (100 limit) so we can export all of them? 

Thxs

**Reply by metmarkosaric** [reply]
Yes, note that the individual CSV files are limited to 300 entries each while page reports are limited to 100 entries each. If you want an even more granular way to export your stats, please take a look at our [stats API](https://plausible.io/docs/stats-api).
